src/WasmComponent.Sdk/build/WasmComponent.Sdk.targets

```diff
@@ -31,13 +31,14 @@
             <WasiPreview1AdapterType>reactor</WasiPreview1AdapterType>
             <WasiPreview1AdapterType Condition="'$(OutputType.ToLower())' == 'exe'">command</WasiPreview1AdapterType>
             <WasiPreview1AdapterPath>$(MSBuildThisFileDirectory)../tools/wasi-wasm/wasi_snapshot_preview1.$(WasiPreview1AdapterType).wasm</WasiPreview1AdapterPath>
-            <NativeComponentBinary>$(NativeOutputPath)$(TargetName).component.wasm</NativeComponentBinary>
+            <!-- NativeComponentBinary should have a filename in kebab-case -->  
+            <NativeComponentBinary>$(NativeOutputPath)$(TargetName.ToLower().Replace(" ", "-"))-component.wasm</NativeComponentBinary>
         </PropertyGroup>
     </Target>
 
     <Target Name="PrepareWasmSdks" BeforeTargets="CheckWasmSdks" DependsOnTargets="ObtainWasiSdk; ObtainEmscripten">
         <PropertyGroup>
-            <EmSdk>$(EmscriptenRoot)</EmSdk>
+            <EmSdk Condition="'$(EmSdk)' == ''">$(EmscriptenRoot)</EmSdk>
```

Comment:
I wonder if this change might be what's breaking the build.

Reply:
That's curious, because it wasn't breaking on my machine. Though I do have the EmSdk environment variable set on my machine. And I had to use this condition because it was otherwise broken.

Reply:
But since the build is complaining about an error with EmSdk, I will revert this change.